packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -138,8 +138,14 @@ export const messages = {
   force_lightning_interface_create_text: 'SFDX: Create Aura Interface',
   force_function_create_text: 'SFDX: Create Function',
   force_function_start_text: 'SFDX: Start Function',
+  force_function_start_warning_no_toml:
+    'No function.toml found. Create function.toml or intialize a new function folder using SFDX: Create Function.',
```

Comment:
This is an edge case that only happens when you have created a function folder, deleted function.toml and try to start function there

Reply:
Project is missing the function.toml file. Create function.toml or create a new function using SFDX: Create Function.

@danyusf - assuming we don't doc this TOML file and how to create it - if we do, then we can leave this message as-is. But if not, would it be better to say to add it back? Or just say create a new function?

Reply:
We will doc the TOML file (we already have it doc'd in the dev preview docs, and at GA will have it in a DSC dev guide). Going forward the scenarios where they have a Function project but no TOML file will be rare (if they use DX to create the project it creates a default TOML for you). I would say _depending on the situation_, a dev might want to just add the TOML file to an existing project they've been working on previously, or start over, so the proposed error message seems reasonable.

Reply:
No function.toml found. Create a function.toml, or create a new function using SFDX: Create Function.